web/bind.go

```diff
@@ -19,8 +19,13 @@ type (
 
 // Bind implements the `Binder#Bind` function.
 func (b *DefaultBinder) Bind(i interface{}, c Context) (err error) {
-	defer dumpError(err, c.Logger(), *c.Request())
 	req := c.Request()
+	dump, _ := httputil.DumpRequest(req, true)
```

Comment:
mb?
```
	defer func(e error){
		dumpError(e, c.Logger(), req)
	}(err)
```

Reply:
defer works with empty req.Body...

Reply:
sorry
```
	defer func(e error){
		dumpError(e, c.Logger(), dump)
	}(err)
```

Reply:
are you see difference?
```
	dump, _ := httputil.DumpRequest(req, true)

	defer func(){
		dumpError(err, c.Logger(), dump)
	}()
```

Reply:
what's the point in hiding `err`?

Reply:
no, defer always get err equal NIL

Reply:
you really check this? or only for comment?

Reply:
https://play.golang.org/p/nloLcBJcHZo

Reply:
@jenchik https://play.golang.org/p/1c1J2WFrsVZ

check your self pls...

Reply:
ok